service.subsmangler/service.py

```diff
@@ -0,0 +1,1742 @@
+import codecs
```

Comment:
It's a bad practice to keep all the logic on the service entry point. Can you simply move this to the lib folder and create e new simple script that just imports this and starts the service loop?

Reply:
will do